LOCALIZE.md

```diff
@@ -121,14 +121,20 @@ Every `para` has a corresponding _.m4a_ recorded audio file and an accompanying
 …
 ```
 
-Each word within a `para` is also individually recorded, along with a version split into syllables for lower-level stories.
-Use [PhraseAnal](https://github.com/XPRIZE/GLEXP-Team-onebillion-PhraseAnal), to assist with generating _.etpa_ files from audio files.
+Each word within a `para` is also individually recorded, along with a version split into syllables for lower-level stories.  
+
+The English transcripts for stories are stored in `assets/transcripts/oc-reading/xr-[id]/english`. Create transcripts for local language and store them in `assets/transcripts/oc-reading/xr-[id]/LOCALE`. Generate audio files in local language and accent from these locale-specific transcripts by using [Amazon Polly](https://docs.aws.amazon.com/polly/latest/dg/what-is.html).  
+**Note:** Check for the languages supported by Amazon Polly [here](https://docs.aws.amazon.com/polly/latest/dg/voicelist.html).
+
+After the audio files have been generated, use [Amazon Polly's Speech Marks](https://docs.aws.amazon.com/polly/latest/dg/speechmarkexamples.html), to assist with generating _.txt_ files from transcripts. Create a custom script to generate _.etpa_ files from the text files generated by Amazon Polly.
```

Comment:
Can you commit this custom script to GitHub?

How do you generate the _.txt_ file.

Reply:
Will raise a separate PR for the script and specific instructions to run the script.